account_invoice_report_picking_customer_note/readme/USAGE.rst

```diff
@@ -0,0 +1,29 @@
+This module assumes you have configured and are using sale_stock_picking_note 
+as described in its README (i.e., the Picking Customer Comments field is 
+available on the Delivery Address contact —Sales & Purchases tab— and on 
+the Sales Order —Other Information tab—).
+
+**Case 1 — Single Sales Order --> Single Invoice**
+#. On the Sales Order, open the Other Information tab and set 
+Picking Customer Comments (technical name: picking_customer_note).
+#. When you create/print the invoice from this SO, the invoice report will 
+display that Picking Customer Comments value from the sales order.
```

Comment:
The formatting of the text isn't right. Take a look at the result... (use markdown as it probably will be easier)

<img width="1040" height="680" alt="Image" src="https://github.com/user-attachments/assets/fba6218a-4b1f-41e4-b45f-6423bd081cb0" />